lblod/app-digitaal-loket
# Add person and account fields to the public graph

## Issue Number

DL-5511

## Issue Description

Searching for `Zorgband Leie en Schelde` on the mock-login frontend will not reveal this organization even though it is active and functioning; this happens on the development, QA and production environments.

## How to Test

Run the PR migration and restart the resource and cache containers:
```
drc restart migrations
```
```
drc restart resource cache
``` 
and search for `Zorgband Leie en Schelde`; the organization should show up now.
